apps/condo/lang/en/en.json

```diff
@@ -1547,6 +1547,8 @@
   "api.banking.createBankAccountRequest.INCORRECT_PROPERTY_ID": "Incorrect propertyId was provided. Please check that this id related to passed organizationId",
   "api.incident.WORK_FINISH_EARLY_THAN_WORK_START": "The end of the works cannot be earlier than the start",
   "api.newsItem.EMPTY_VALID_BEFORE_DATE": "Validity date is required",
+  "api.newsItem.EDIT_DENIED_ALREADY_SENT": "You cannot edit the news item which already been sent",
```

Comment:
Is this translation used somewhere?

Reply:
I'll use it later. Thanks for your attentiveness.